data_structures/src/chain.rs

```diff
@@ -635,6 +635,7 @@ impl Input {
 pub struct ValueTransferOutput {
     pub pkh: PublicKeyHash,
     pub value: u64,
+    pub time_lock: u64,
```

Comment:
```suggestion
    /// Do not accept transactions which reference this transaction until timestamp
    pub time_lock: u64,
```

Same for data requests

Reply:
For VTTs, the most accurate description would be:
> The value attached to a time-locked output cannot be spent before the specified timestamp. That is, they cannot be used as an input in any transaction of a subsequent block for an epoch whose opening timestamp predates the time lock.